### PR TITLE
Build and push images to dockerhub from circle #49

### PR DESCRIPTION
**Pull request by jaredkerim**

### CODE REVIEW

**Comment by relud** [comment]
lgtm, if it works

File: circle.yml [hunk 1]
```diff
@@ -2,6 +2,46 @@ machine:
...
+      - "[ ! -z $DOCKERHUB_REPO ]"
+      - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+      - "docker tag app:build ${DOCKERHUB_REPO}:latest"
+      - "docker push ${DOCKERHUB_REPO}:latest"
+
+  hub_releases:
+    # push all tags
+    tag: /.*/
+    commands:
+      - "[ ! -z $DOCKERHUB_REPO ]"
```
Comment (relud) [inline circle.yml]: you should use `-n "$DOCKERHUB_REPO"` not `! -z $DOCKERHUB_REPO`. same on line 33 above.

Comment (jaredkerim) [inline circle.yml]: Is that special to my case or should that also change in 

https://github.com/mozilla-services/Dockerflow/blob/master/circle.yml#L41

File: circle.yml [hunk 2]
```diff
@@ -2,6 +2,46 @@ machine:
...
 test:
   override:
     - make test
+
+# appropriately tag and push the container to dockerhub
+deployment:
+  hub_latest:
+    branch: "master"
+    commands:
+      - "[ -n \"$DOCKERHUB_REPO\" ]"
```
Comment (jaredkerim) [inline circle.yml]: @relud is this correct?

Comment (relud) [inline circle.yml]: yes

**Comment by relud** [comment]
r+

**Pull request merged**